channels/mx.m3u

```diff
@@ -3,29 +3,29 @@
 http://217.182.77.27/live/alcarriatv-livestream.m3u8
-#EXTINF:-1 tvg-id="XHTITTDT.mx" tvg-name="Azteca 7" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/20551-91383.png" group-title="",Azteca 7 (XHTIT-TDT)
+#EXTINF:-1 tvg-id="XHTITTDT.mx" tvg-name="Azteca 7" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/20551-91383.png" group-title="Local",Azteca 7 (XHTIT-TDT)
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtit-tdt.m3u8
-#EXTINF:-1 tvg-id="XHASTDT.mx" tvg-name="Azteca America" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/en/2/26/AZTECA_33_current_logo.png" group-title="",Azteca America (XHAS-TDT)
+#EXTINF:-1 tvg-id="XHASTDT.mx" tvg-name="Azteca America" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/en/2/26/AZTECA_33_current_logo.png" group-title="Local",Azteca America (XHAS-TDT)
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhas-tdt.m3u8
-#EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-name="Azteca Uno" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="",Azteca Uno (XHJK-TDT)
+#EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-name="Azteca Uno" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="Local",Azteca Uno (XHJK-TDT)
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhjk-tdt.m3u8
-#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-name="Canal 10 Cancún" tvg-country="MX" tvg-language="" tvg-logo="" group-title="",Canal 10 Cancún
+#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-name="Canal 10 Cancún" tvg-country="MX" tvg-language="" tvg-logo="" group-title="Local",Canal 10 Cancún
 http://stream2.dynalias.com:1935/live/tvlive1/chunklist_w1211325251.m3u8
-#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-name="Canal 10 Cancún" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://s3-us-west-1.amazonaws.com/canal10/photos/96800/original.jpg" group-title="",Canal 10 Cancún (720p)
+#EXTINF:-1 tvg-id="Canal10Cancun.mx" tvg-name="Canal 10 Cancún" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://s3-us-west-1.amazonaws.com/canal10/photos/96800/original.jpg" group-title="Local",Canal 10 Cancún (720p)
 http://stream2.dynalias.com:1935/live/tvlive1/playlist.m3u8
-#EXTINF:-1 tvg-id="XEWTTDT.mx" tvg-name="Canal 12 - Televisa" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/12.1.png" group-title="",Canal 12 - Televisa (XETW-TDT)
+#EXTINF:-1 tvg-id="XEWTTDT.mx" tvg-name="Canal 12 - Televisa" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/12.1.png" group-title="Local",Canal 12 - Televisa (XETW-TDT)
```

Comment:
Ehh... that's quite touchy.

Again, it's from a regional transmitter, they *do* stream regional/local programs, according to https://en.wikipedia.org/wiki/XEWT-TDT, but they do reconnect after with the national programming...

Not sure what to say here.

Reply:
I think it's fine to consider it as a local if local programming is being broadcasted on it

Reply:
I guess you're right. No need to touch that one.